### PR TITLE
Prepare 1.0

### PR DESCRIPTION
**Pull request by arturbosch**

### CODE REVIEW

**Comment by arturbosch** [comment]
Note: this pr will crash until the release it out.

**Comment by 3flex** [comment]
Maybe one final RC release then go for 1.0.0 in a week or two, only allowing bug fixes between now and then?

**Comment by arturbosch** [comment]
I've tested the new release on our analysis projects and all is fine.
I would go with with this release. There is no shame to do some hotfixes for 1.0.1 or 1.0.2 :).
Lets get this out. Rdy when you are @3flex @schalkms

File: docs/_posts/2019-08-04-calling-it-one-zero.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+There are **2516** commits, **52** releases on GitHub and a total **93** contributors by the time of writing.
+**20** out of the 93 authors contributed once or more in the last three months.
+
+![numbers](/images/one-zero/3-bintray-absolute.png)
+
+**~780k** downloads in the last **30 days** is a pretty high number ... three months ago it was like **500k**.
+One can clearly see when weekends are ;).
+
+This does however not mean "1 download = 1 user". There are like eight detekt modules each with a jar and pom which needs to be downloaded etc.
+Most of the downloads should be coming from CI though it is pretty hard to calculate the number of users detekt actually has.
```
Comment (schalkms) [inline docs/_posts/2019-08-04-calling-it-one-zero.md]: though. It is pretty...

File: docs/pages/changelog 1.x.x.md [hunk 2]
```diff
@@ -6,11 +6,49 @@ permalink: changelog.html
...
-<!--
-#### Coming up
+#### 1.0.0
 
 ##### Migration
--->
-#### 1.0.0
 
-*Coming soon*
+- Gradle Plugin: removed report consolidation from. It was flawed and some users were stuck with RC14. It will be replaced in a further version.
```
Comment (schalkms) [inline docs/pages/changelog 1.x.x.md]: From what?

File: docs/pages/changelog 1.x.x.md [hunk 3]
```diff
@@ -6,11 +6,49 @@ permalink: changelog.html
...
+- Gradle Plugin: `autoCorrect` property is not allowed on the detekt extension. No need to create a new task anymore.
+- Formatting: updated to KtLint 0.34.2 which removed two rules `NoItParamInMultilineLambda` and `SpacingAroundUnaryOperators`.
```
Comment (schalkms) [inline docs/pages/changelog 1.x.x.md]: The two rules

File: docs/_posts/2019-08-04-calling-it-one-zero.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+- RC09 was released in Sep 2018
+- RC10 was released in Nov 2018
+- RC11 was released in Nov 2018
+- RC12 was released in Dec 2018
+- RC14 was released in Feb 2019
+- RC15 was released in Jun 2019
+- RC16 was released in Jun 2019
+
+**65%** of users are on a version published in 2019. But we clearly lost some users in the older versions due to breaking changes in RC13 and RC15.
+I'm excited to see how many users will jump to the 1.x.x release train and how this numbers will look like.
```
Comment (cortinico) [inline docs/_posts/2019-08-04-calling-it-one-zero.md]: nit:

```suggestion
I'm excited to see how many users will jump to the 1.x.x release train and how these numbers will look like.
```

**Comment by 3flex** [comment]
In the news post it would be good to have a brief intro to detekt, also noteworthy features added during its development - consider those who might see this news on Reddit for example and haven't used the tool before, or used it a while ago but haven't kept up with development. [Flutter 1.0](https://developers.googleblog.com/2018/12/flutter-10-googles-portable-ui-toolkit.html) announcement is a good example. Can do something much more brief but that structure is pretty typical of a 1.0 announcement.

Also what 1.0.0 means - now following SemVer? Also which parts are considered public API and therefore respect SemVer? I think it's mentioned in an issue somewhere but should be part of official docs and included in the announcement.

File: docs/_posts/2019-08-04-calling-it-one-zero.md (hunk 4 above)
Comment (arturbosch) [reply docs/_posts/2019-08-04-calling-it-one-zero.md]: Thanks, and I gonna change this to "we" :)

**Comment by arturbosch** [comment]
Thanks for feedback and spell checking!

**Comment by schalkms** [comment]
@arturbosch unfortunately the images are not referenced correctly on the homepage.

I refer to the following news page.
https://arturbosch.github.io/detekt/calling-it-one-zero.html

**Comment by arturbosch** [comment]
Thanks!

**Pull request merged**